src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -65,41 +64,103 @@ class Widget_List extends Widget_Abstract {
 	];
 
 	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
 	 * {@inheritDoc}
-	 *
-	 * @var array<string,mixed>
 	 */
-	protected $validate_arguments_map = [
-		'should_manage_url'    => 'tribe_is_truthy',
-		'no_upcoming_events'   => 'tribe_is_truthy',
-		'featured_events_only' => 'tribe_is_truthy',
-		'jsonld_enable'        => 'tribe_is_truthy',
-	];
+	public function get_arguments() {
+		$arguments = $this->arguments;
+
+		$arguments['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
+		// @todo update name once this widget is ready to replace the existing list widget.
+		$arguments['name']                          = esc_html__( 'Events List V2', 'the-events-calendar' );
+		$arguments['widget_options']['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
+
+		// Setup default title.
+		$arguments['title'] = __( 'Upcoming Events', 'the-events-calendar' );
+
+		// Setup admin fields.
+		$arguments['admin_fields'] = $this->get_admin_fields();
+
+		// Add the Widget to the arguments to pass to the admin template.
+		$arguments['widget_obj'] = $this;
+
+		$arguments = wp_parse_args( $arguments, $this->get_default_arguments() );
+
+		return $this->filter_arguments( $arguments );
+	}
 
 	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
 	 * {@inheritDoc}
 	 */
-	public function setup() {
-		$this->setup_view();
+	public function update( $new_instance, $old_instance ) {
```

Comment:
Added a new filter here.

Reply:
I think this reply was for my next comment, I cannot see an abstract `update` method in the parent Widget class in Common.

Reply:
Correct on the comment. However, the WP_Widget class which we extend already has a Public Update method. This prevents us from having an abstract method in our abstract classes. I can add a public method to the abstract instead. For now I would like to leave this coding as is and in the next widget ( mini calendar ) we refactor the update system into the abstract class so it can handle all the widgets.